flutter-idea/build.gradle

```diff
@@ -33,6 +33,8 @@ intellij {
 dependencies {
   testCompile "org.powermock:powermock-api-mockito2:2.0.0"
   testCompile "org.powermock:powermock-module-junit4:2.0.0"
+  implementation fileTree(dir: "$project.rootDir/lib/jxbrowser",
```

Comment:
Is this change leftover from working on jxbrowser stuff? Or is it necessary for dealing with the jxbrowser files that are in your directory but not committed?